docuploader/protos/metadata_pb2.py

```diff
@@ -35,6 +34,7 @@
     filename=None,
     file=DESCRIPTOR,
     containing_type=None,
+    create_key=_descriptor._internal_create_key,
```

Comment:
This was added in protobuf 3.12.0 so the lower bound probably needs to be increased:

https://github.com/googleapis/docuploader/blob/ff280bfc114ebaa15d820ad04d2bc06eda013843/setup.py#L25

Reply:
Done.